package.json

```diff
@@ -107,7 +107,7 @@
     "react-native-pdf": "^6.6.2",
     "react-native-performance": "^4.0.0",
     "react-native-permissions": "^3.0.1",
-    "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#84ee97dec11c2e65609511eb5a757d61bbeeab79",
+    "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#683fd45e3ca18054e2d3ceb319e0cf595a1e26ed",
```

Comment:
We have to ensure this SHA1 is up-to-date at the end of the review.

Let's resolve this thread last.

Reply:
I used the path to the `Expensify` fork, but that might've been a bad idea. Fixed.